### ISSUE TITLE
Test suite build failure

### ISSUE THREAD

**Issue by snoyberg** [docopt/docopt.hs#29]
```
[11 of 11] Compiling Main             ( test/LangAgnosticTests.hs, dist/build/tests/tests-tmp/Main.o )

test/LangAgnosticTests.hs:149:28: error:
    • Overlapping instances for ToJSON (Map Option ArgValue)
        arising from a use of ‘toJSON’
      Matching instances:
        instance (ToJSON v, ToJSONKey k) => ToJSON (Map k v)
          -- Defined in ‘aeson-1.0.2.0:Data.Aeson.Types.ToJSON’
        instance ToJSON (Map Option ArgValue)
          -- Defined at test/LangAgnosticTests.hs:33:10
    • In the expression: toJSON parsedArgs
      In an equation for ‘parsedArgsJSON’:
          parsedArgsJSON = toJSON parsedArgs
      In the expression:
        do { let (cmdline, rawTarget_) = break (== '\n') testcase
                 rawTarget = filter (/= '\n') rawTarget_
                 ....;
             let (parsedArgs, argParseMsg) = ...;
             let parsedArgsJSON = toJSON parsedArgs
                 testCaseEquality = ...
                 ....;
             let testDescLines = ...;
             .... }
```

**Comment by ryanartecona** [comment]
@snoyberg This should be fixed in docopt 0.7.0.5, just uploaded. Apologies for the delay.

Relatedly, I used this as an opportunity to get this library working with stack. Once I did, building against that latest aeson was dead simple. Thanks for that your work to give us better tools!

**Comment by snoyberg** [comment]
Awesome, glad to hear it!
